src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,437 @@
+-   One subject single dynamic scan (pet, blood):
+	[`pet_example3`](https://www.dropbox.com/sh/i0c3iq50id4s0d3/AABSjaczHM1SQnMRvh3fc7fna?dl=0)
+
+Further PET datasets are available from [OpenNeuro](https://openneuro.org).
+
+## PET recording data
+
+As specified above, this extension is relevant for brain PET imaging
+and its associated data, such as blood data.
+In addition, the extension is in accordance with the guidelines for reporting
+and sharing brain PET data (Knudsen et al. 2020, [doi:10.1177/0271678X20905433](https://doi.org/10.1177/0271678X20905433)).
+
+Template:
+
+```Text
+sub-<participant_label>/
+      [ses-<session_label>/]
+        pet/
+        sub-<participant_label>[_ses-<session_label>][_task-<task_label>][_acq-<label>][_rec-<label>][_run-<index>]_pet.nii[.gz]
+        sub-<participant_label>[_ses-<session_label>][_task-<task_label>][_acq-<label>][_rec-<label>][_run-<index>]_pet.json
+```
+
+**Shared MRI data along with PET:**
+To share structural magnetic resonance (MR) images with your PET data,
+please follow the original BIDS specification
+([LINK](https://bids-specification.readthedocs.io/en/stable)).
+Please pay specific attention to whether the MR images have been unwarped to correct for gradient non-linearities. This should be specified in a specific field of the MRI-BIDS .json file named "NonLinearGradientCorrection". For more information please see the appendix regarding cross modality correspondence
+ ([LINK](../99-appendices/10-cross-modality-correspondence.md)).
+
+**Units**: In general, SI units must be used (we refer to [LINK](https://bids-specification.readthedocs.io/en/stable/99-appendices/05-units.html)) and we recommend to use the CMIXF style formatting for SI units, for example, "kBq/mL"
+rather than "kilobecquerel per ml".
+An overview of a common PET experiment (with blood data) can be seen in Figure 1,
+defined on a single time scale relative to a predefined “time zero”
+(which should be defined either to be scan start or injection time;
+please note an exception to this definition is possible if a pharmacological
+within-scan challenge is performed).
+
+![Figure 1](images/PET_scan_overview.svg "Overview of PET recording data")
+
+Figure 1: Overview of a common PET experiment, including blood measurements,
+and defined on a common time scale.
+Note, “time zero” can either be defined as time of injection or scan start,
+and all the PET and blood data should be decay-corrected to this time point.
+Furthermore, although in this example tracer injection coincides with scan start,
+this is not always the case and hence we allow for the flexibility of specifying
+either time of injection or scan start as “time zero”.
+
+PET data belong to the `pet` directory. PET imaging data SHOULD be stored in 4D
+(or 3D if only one volume was acquired) NIfTI files with `_pet` suffix.
+Volumes should be stored in chronological order (the order they were acquired in).
```

Comment:
Shouldn't we prefer `Volumes MUST be stored in chronological order (the order they were acquired in).` 
If we use `Volumes SHOULD be stored in chronological order (the order they were acquired in).`  How do we ensure the ordering --> "FrameTimesStart" in the json file I guess ; does the validator issue a warning making the user aware the order is not chronological?

Reply:
I agree with that change in stating that they MUST be in chronological order. I can't think of a PET example where this isn't the case.  We'll follow up on the validator if this actually doing it. Maybe @rwblair can comment?